tests/unit/FilingHistoryList2.spec.ts

```diff
@@ -108,7 +108,7 @@ filings.forEach((filing: any, index: number) => {
       // expect(item.isFutureEffectiveIaPending).toBeDefined() // FUTURE: test this more specifically
     })
 
-    itIf(isBcompCoa(filing))('BCOMP change of address filing', () => {
```

Comment:
Well, it _should_ taking filing as a parameter, but `isBcompCoa` was updated in a previous commit to not take a parameter because it's not implemented yet.

I'm thinking it would be better to restore `isBcompCoa` and this line and comment them out instead.

Reply:
Reverted, thanks.